docs/index.rst

```diff
@@ -42,10 +41,10 @@ Now you need to set up Raven.js to use your `Sentry DSN
 At this point, Raven is ready to capture any uncaught exception.
 
 Once you have Raven up and running, it is highly recommended to check out :doc:`config`
```

Comment:
feels redundant to say "check out Configuration and Usage for information about configuration and usage", going to ship as-is